test/pa11y.conf.js

```diff
@@ -0,0 +1,62 @@
+const config = {
+
+  // test against WCAG 2.0 AA
+  standard: 'WCAG2AA',
+
+  // only fail on errors
+  level: 'error',
+
+  // wait 500ms before checking
+  wait: 500
+};
+
+/**
+ * Validations that we can safely ignore:
+ */
+const IGNORE_MESSAGES = [
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).